src/main/webapp/scripts/main.js

```diff
@@ -16,5 +16,11 @@ $().ready(() => {
       $(".base").show(200);
       $("#real-body").removeClass("focus");
     });
+
+    $(".active .item").click(function() {
+        $(".item").off("click");
```

Comment:
[nit] should this indentation be four spaces?

Reply:
No, I'll fix it.